Packages/MIES/MIES_Constants.ipf

```diff
@@ -1610,3 +1610,6 @@ Constant PGC_MODE_ASSERT_ON_DISABLED = 0
 Constant PGC_MODE_FORCE_ON_DISABLED  = 1
 Constant PGC_MODE_SKIP_ON_DISABLED   = 2
 /// @}
+
+Constant TP_BASELINE_FRACTION_LOW  = 0.25
+Constant TP_BASELINE_FRACTION_HIGH = 0.49
```

Comment:
@t-b why is it limited to 0.49?

Reply:
@timjarsky We limit the baseline % since forever to max 49%, see 

```
+    SetVariable SetVar_DataAcq_TPBaselinePerc,limits={25,49,1},value= _NUM:25
```

from 1f7cfa65 (Make the baseline length of the testpulse configurable, 2015-07-04).

50% would be no pulse.